src/lib/components/Pagination/Pagination.js

```diff
@@ -13,10 +13,24 @@ import Overridable from 'react-overridable';
 import { ShouldRender } from '../ShouldRender';
 import { buildUID } from '../../util';
 
+const defaultOptions = {
+  boundaryRangeCount: 1,
+  siblingRangeCount: 1,
+  showEllipsis: true,
+  showFirst: true,
+  showLast: true,
+  showPrev: true,
+  showNext: true,
+  size: 'large',
+};
+
 class Pagination extends Component {
   constructor(props) {
     super(props);
     this.updateQueryPage = props.updateQueryPage;
+    this.options = props.options
+      ? { ...defaultOptions, ...props.options }
```

Comment:
This change was made because it's easier to override specific options e.g `options={size: 'small'}` without having to provide all the other keys.